packages/cli/src/lib/merge/strategy.js

```diff
@@ -0,0 +1,94 @@
+import * as t from '@babel/types'
+
+import { fillUnique, nodeIs, overlap, pushUnique, sieve } from './algorithms'
+
+export const mergeUtility = {
+  _insertAfter: (base, ...exts) => {
+    base.insertAfter(...exts)
+  },
+  keepBoth: (base, ext) => {
+    mergeUtility._insertAfter(base, ext.node)
+    base.skip()
+  },
+  keepBothStatementParents: (base, ext) => {
+    mergeUtility._insertAfter(
+      base.getStatementParent(),
+      ext.getStatementParent().node
+    )
+    base.skip()
+  },
+  keepBase: (_base, _ext) => {},
+  keepExtension: (base, ext) => {
+    base.replaceWith(ext)
+    base.skip()
+  },
+}
+
```

Comment:
These are super neatly written functions - I love it, but babel transforms are notoriously difficult to read. 

Maybe add a few unit tests here to demonstrate what it does?